app/views/games/new.html.erb

```diff
@@ -28,5 +28,7 @@
   <%= f.label :description, "description" %>
   <%= f.text_area :description %>
 
+  <%= f.hidden_field :user_id, :value => current_user.id %>
```

Comment:
Do not pass the `user_id` through the form. This is easy to manipulate on the client and would allow them to set the ID to some arbitrary value. Instead, rely on the `current_user` helper within your controller to assign the user:

```ruby
def create
  @game = Game.new(game_params)
  @game.user = current_user

  # ...
end
```